src/const.js

```diff
@@ -20,6 +20,7 @@
  * at runtime from the About dialog for additional information.
  *****************************************************************************/
 
+export const EVENTS_OBJECT_TYPE = 'events';
```

Comment:
can we add a unique id for this type, to avoid clash with openmct. Looking at other types, maybe we can use yamcs.events